styles/variables-base.less

```diff
@@ -447,6 +447,7 @@
 @agate-popup-border: 0 solid transparent;
 @agate-popup-padding: (@agate-component-spacing * 2);
 @agate-popup-closebutton-title-padding: 0 0 0 (@agate-button-small-height);
+@agate-popup-closebutton-centered-title-padding: @agate-popup-closebutton-title-padding;
```

Comment:
To fix for all skins, How about 
```
@agate-popup-closebutton-centered-title-padding: 0 (@agate-button-small-height);
```
and remove "@agate-popup-closebutton-centered-title-padding" in silicon.